galasa-managers-parent/galasa-managers-testingtools-parent/dev.galasa.jmeter.manager.ivt/src/main/java/dev/galasa/jmeter/manager/ivt/JMeterManagerIVT.java

```diff
@@ -0,0 +1,69 @@
+package dev.galasa.jmeter.manager.ivt;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.InputStream;
+
+import org.apache.commons.logging.Log;
+
+import dev.galasa.Test;
+import dev.galasa.artifact.ArtifactManager;
+import dev.galasa.artifact.IArtifactManager;
+import dev.galasa.artifact.IBundleResources;
+import dev.galasa.artifact.TestBundleResourceException;
+import dev.galasa.core.manager.Logger;
+import dev.galasa.docker.DockerContainer;
+import dev.galasa.docker.IDockerContainer;
+import dev.galasa.jmeter.IJMeterSession;
+import dev.galasa.jmeter.JMeterSession;
+import dev.galasa.jmeter.JMeterManagerException;
+
+/*
+ * Licensed Materials - Property of IBM
+ * 
+ * (c) Copyright IBM Corp. 2020.
+ */
+
+ @Test
+ public class JMeterManagerIVT {
+
+   @Logger
```

Comment:
Really small niggle (Sorry) can this be indented to match the rest of the variables please